### DISCUSSION TITLE
Throw Err node:internal/modules/cjs/loader:998 Error: Cannot find module examples/browser/src-gen/backend/main.js

### DISCUSSION THREAD

**Discussion by jittygitty** [eclipse-theia/theia#11947] [Q&A] [answered]
### Bug Description:
Not SURE if this is related to the TWO years ago issue of https://github.com/eclipse-theia/theia/issues/7568

I can not get it to install/run properly:
node:internal/modules/cjs/loader:998
throw err;
^
Error: Cannot find module '/home/user/Theia/theia-1.32.0/examples/browser/src-gen/backend/main.js'
at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
at Module._load (node:internal/modules/cjs/loader:841:27)
at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
at node:internal/main/run_main_module:23:47 {
code: 'MODULE_NOT_FOUND',
requireStack: []
}
Node.js v18.12.1
Done in 2.50s.

And yes, there's no src-gen folder in there, apparently yarn install did not create it although it finished successfully.

### Steps to Reproduce:

yarn install
yearn start

### Additional Information

- Operating System: Debian 11
- Theia Version: 1.32.0

**Comment by msujew** [comment, marked as answer]
Hey @jittygitty,

I'm not sure what your setup is, but depending on how your set up your project, you might need to run `yarn build` before `yarn start`. `yarn install` will usually just install dependencies and not really build your project.

Edit: I just saw that you're trying to build our examples and not your own project. Our repo definitely requires to run `yarn build` before `yarn start`.

**Comment by jittygitty** [comment]
Oh wow, i feel stupid, thanks! (i think javascript hates me cuz i gave it the cold shoulder "long ago" :(  ...
